src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -339,6 +368,160 @@ public OptionData setChannelTypes(@Nonnull Collection<ChannelType> channelTypes)
         return this;
     }
 
+    /**
+     * Configure the minimal value which can be provided for this option.
+     *
+     * @param  value
+     *         The minimal value which can be provided for this option.
+     * @throws IllegalArgumentException
+     *         If any of the following checks fail
+     *         <ul>
+     *             <li>{@link OptionType type of this option} is {@link OptionType#INTEGER INTEGER}</li>
+     *             <li>{@code value} is larger than or equal to {@link OptionData#MIN_NEGATIVE_NUMBER MIN_NEGATIVE_NUMBER}</li>
+     *         </ul>
+     *
+     * @return The OptionData instance, for chaining
+     */
+    @Nonnull
+    public OptionData setMinValue(long value)
+    {
+        if (type != OptionType.INTEGER)
```

Comment:
I mean, a long value is also a valid double value in our case since its restricted to 2^53 regardless and we should allow people to write code like `new OptionData(DOUBLE, "volume", "Set the music voluem").setMaxValue(1)`